q2lint/_main.py

```diff
@@ -7,8 +7,28 @@
 # ----------------------------------------------------------------------------
 
 import argparse
+import datetime
 import pathlib
 import sys
+import re
+
+
+YEAR_PLACEHOLDER = "COPYRIGHT_YEARS"  # chosen to avoid re.escape replacement
+
+
+def check_license(license, reference, copyright_idx):
+    for idx, (line, ref) in enumerate(zip(license.splitlines(),
```

Comment:
I think we should use `itertools.zip_longest` so that the loop doesn't terminate early if one license is shorter than the other.